### PR TITLE
Add SPU usage for program dump, remove welcome dialog keyboard-shortcut

### LINKED ISSUE #15839: [Feature request] A simple "Press CTRL+F1 for help" in the debugger
### Quick summary

I was looking for info about the debugger in its code (literally no info is present in the debugger itself), only to find that it already has an inbuild small help page with shortcuts.
Seriously?
Also, even in that little help window it says that to bring it up, you need to press F1, but you actually need to press ctrl+F1.

### Details

_No response_

### PR DESCRIPTION
**Pull request by elad335**
* When SPU Debug is option is enabled, percent of usage of SPU programs is added to the log when dumping SPU code.
* Remove welcome dialog shortcut and add description in debugger that F1 invokes the help dialog. Fixes #15839
* Fix some minor bugs regarding potential overflows.

### CODE REVIEW

File: rpcs3/Emu/System.cpp [hunk 1]
```diff
@@ -3489,6 +3491,7 @@ void Emulator::Kill(bool allow_autoexit, bool savestate, savestate_stage* save_s
 				log_buffer.resize(spu_log.read(log_buffer.data(), log_buffer.size()));
 				spu_log.close();
 
+				// , jjHqNzSUs9dN03TqogfukwARX2bY) ==========
```
Comment (Megamouse) [inline rpcs3/Emu/System.cpp:3494]: What is this supposed to be?

File: rpcs3/rpcs3qt/shortcut_settings.cpp [hunk 2]
```diff
@@ -44,7 +43,6 @@ shortcut_settings::shortcut_settings()
 		{ shortcut::mw_toggle_fullscreen, shortcut_info{ "main_window_toggle_fullscreen", tr("Toggle Fullscreen"), "Alt+Return", shortcut_handler_id::main_window } },
 		{ shortcut::mw_exit_fullscreen, shortcut_info{ "main_window_exit_fullscreen", tr("Exit Fullscreen"), "Esc", shortcut_handler_id::main_window } },
 		{ shortcut::mw_refresh, shortcut_info{ "main_window_refresh", tr("Refresh"), "Ctrl+F5", shortcut_handler_id::main_window } },
-		{ shortcut::mw_welcome_dialog, shortcut_info{ "main_window_welcome_dialog", tr("Show Welcome Dialog"), "F1", shortcut_handler_id::main_window } },
```
Comment (Megamouse) [inline rpcs3/rpcs3qt/shortcut_settings.cpp:47]: Why are you removing this?

Comment (elad335) [reply rpcs3/rpcs3qt/shortcut_settings.cpp:47]: it conflicts with debugget help shortcut, I also don't think there is a need for a shortcut to the welcome dialog. It is a welcome dialog after all, should be displayed once and if the user wants to see it again it's in the menu.

**Comment by Hasster1** [comment]
Awesome job, Thanks!

**Pull request merged**